framework/decode/vulkan_replay_consumer_base.cpp

```diff
@@ -4440,6 +4445,27 @@ VkResult VulkanReplayConsumerBase::OverrideAllocateMemory(
                                                replay_memory,
                                                &allocator_data);
         }
+        else if (import_fd_found)
+        {
+            VkMemoryAllocateInfo modified_allocate_info = (*replay_allocate_info);
+
+            VkBaseOutStructure* current_struct = reinterpret_cast<VkBaseOutStructure*>(&modified_allocate_info);
+            while (current_struct->pNext)
+            {
+                if (current_struct->pNext->sType == VK_STRUCTURE_TYPE_IMPORT_MEMORY_FD_INFO_KHR)
```

Comment:
So I did this and I got a crash on binding the memory to either an image or a buffer because the memory formats and indices didn't match.  Check out the top commit on [my fork](https://github.com/MarkY-LunarG/gfxreconstruct/) branch `marky_test_remove_fd`.